### PR TITLE
checks for course info widgets via MutationObserver in the case where…

### PR DESCRIPTION
**Pull request by elliottyates**
… they are not rendered prior to our global.js executing

### CODE REVIEW

**Comment by sapnamysore** [comment]
@sapnamysore

File: js/iframe_resizer.js [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+          }
       });
+
+      var moConfig = {
+        childList: true,
+        subtree: true
+      };
+
+      courseInfoObserver.observe(document.getElementById('content'), moConfig);
     }
```
Comment (sapnamysore) [inline js/iframe_resizer.js:33]: The target of the observer  is the  'content' div - so basically the page contents in the middle(excluding left navigation and the right)?

**Comment by sapnamysore** [comment]
Looks good :+1:

File: js/iframe_resizer.js (hunk 1 above)
Comment (elliottyates) [inline js/iframe_resizer.js:33]: Yep! Preliminary testing indicates this should always be there before react builds the page further. Just reduces the number of mutation triggers.

**Pull request merged**